### ISSUE TITLE
Is this ready?

### ISSUE THREAD

**Issue by bestfriendai** [BillSchumacher/Auto-Vicuna#1]
Can I run this with AutoGPT that uses Vicuna and can browse the web?

**Comment by BillSchumacher** [comment]
No, you can chat with a blank conversation template at this moment.

**Comment by BillSchumacher** [comment]
Or fork it and provide a custom conversation that's probably more useful.

**Comment by venkatapathy** [comment]
Can you elaborate on how to add this to AutoGPT? I installed auto_vicuna and able to chat like how I can do with fastchat. But unsure on how to integrate into AutoGPT or run AutoVicuna like AutoGPT.
